arangod/IResearch/IResearchDataStore.cpp

```diff
@@ -568,6 +569,7 @@ IResearchDataStore::Snapshot IResearchDataStore::snapshot() const {
         << id() << "'";
     return {};  // return an empty reader
   }
+
```

Comment:
Should we throw some exception in case of failed link? Or check for failed in the IResearchInvertedIndex::resetFilter